util.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+// Node.js TextDecoder/TextEncoder
+import {TextDecoder, TextEncoder} from 'util';
+import base64url from 'base64url-universal';
```

Comment:
It seems like the only difference between browser/node here is where you get `TextEncoder` and `TextDecoder` from. Can we make a text-codec.js and text-codec-browser.js file instead and just export those from there and then we can pull them into a common `util` file? Would be easier on maintenance.

Reply:
The interesting thing is, we keep repeating this logic (about the TextEncoders) in a bunch of repos. I wonder if it makes sense to like make a text-encoder-universal package or something.

Reply:
Yes, a simple package that exports the codecs per platform would be good.  I thought about this long ago, but figured it wasn't worth the trouble at the time.  But seems the repeated code has hit critical mass to justify it now.  At the time I was thinking "textcodec-universal", but maybe "text-codec-universal" is better?  "encoder" seems wrong in name since it has a decoder too.

Reply:
Resolved.